src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,199 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+import duke.DukeDate;
+/**
+ * Represents the list of tasks the user has. Stores the tasks in the form of <code>Task</code> objects.
+ */
+public class TaskList {
+    private ArrayList<Task> userList;
+    private int listSize = 0;
+
+    /**
+     * Creates a new TaskList.
+     */
+    public TaskList() {
+        userList = new ArrayList<Task>(100);
```

Comment:
Might want to declare 100 as a constant (e.g. `private static final int MAX_ARRAY_SIZE = 100`)